connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala

```diff
@@ -4,18 +4,18 @@ import scala.concurrent.duration._
 
 object KustoConstants {
   // Setting high value to have no timeout on Await commands
-  val defaultWaitingIntervalLongRunning: String = (2 days).toSeconds.toString
-  val defaultPeriodicSamplePeriod: FiniteDuration = 1 seconds
-  val defaultIngestionTaskTime: FiniteDuration = 20 seconds
-  val clientName: String = KustoDataSourceUtils.clientName
-  val defaultBufferSize: Int = 16 * 1024
-  val storageExpiryMinutes: Int = 120
-  val sparkSettingsRefreshMinutes: Int = 120
-  val oneKilo: Int = 1024
-  val oneMega: Int = oneKilo * 1024
-  val oneGiga: Int = oneMega * 1024
+  val DefaultWaitingIntervalLongRunning: String = (2 days).toSeconds.toString
+  val DefaultPeriodicSamplePeriod: FiniteDuration = 1 seconds
+  val DefaultIngestionTaskTime: FiniteDuration = 20 seconds
+  val ClientName: String = KustoDataSourceUtils.clientName
+  val DefaultBufferSize: Int = 16 * 1024
+  val StorageExpiryMinutes: Int = 120
+  val SparkSettingsRefreshMinutes: Int = 120
+  val OneKilo: Int = 1024
```

Comment:
nit: units are left out. Recommend including the unit (e.g. OneKilobit)

Reply:
OneKilobite ...